modules/cli/src/main/java/org/apache/ignite/cli/commands/node/status/NodeStatusSubCommand.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.ignite.cli.commands.node.status;
+
+import jakarta.inject.Inject;
+import jakarta.inject.Singleton;
+import java.util.concurrent.Callable;
+import org.apache.ignite.cli.call.node.status.NodeStatusCall;
+import org.apache.ignite.cli.commands.BaseCommand;
+import org.apache.ignite.cli.commands.decorators.NodeStatusDecorator;
+import org.apache.ignite.cli.core.call.CallExecutionPipeline;
+import org.apache.ignite.cli.core.call.StatusCallInput;
+import picocli.CommandLine.Command;
+import picocli.CommandLine.Option;
+
+/**
+ * Display the node status.
+ */
+@Command(name = "status",
+        description = "Prints status of the node.")
+@Singleton
+public class NodeStatusSubCommand extends BaseCommand implements Callable<Integer> {
+
+    /**
+     * Node url option.
+     */
+    @SuppressWarnings("PMD.UnusedPrivateField")
+    @Option(
+            names = {"--node-url"}, description = "Url to node.",
+            descriptionKey = "ignite.node-url", defaultValue = "http://localhost:10300"
```

Comment:
Default value not needed anymore after IGNITE-17130

Reply:
in case IGNITE-17130 is merged, I wil rebase